lsp-fiber/src/rpc.ml

```diff
@@ -2,6 +2,28 @@ open Import
 open Jsonrpc
 module Session = Jsonrpc_fiber.Make (Fiber_io)
 
+module Reply = struct
+  type 'r t =
+    | Now of ('r, Jsonrpc.Response.Error.t) result
```

Comment:
Would be nice to extract `('r, Jsonrpc.Response.Error.t) result` in its own type: it comes back regularly, so it would certainly improve readability.

Reply:
It can later be moved to `Import.Or_error`

Reply:
Actually, I'm thinking of dropping these result types altogether. We already raise in many places, might as well stick to one way of doing things.